src/sbd-md.c

```diff
@@ -1049,7 +1049,7 @@ static int servant_check_timeout_inconsistent(struct sector_header_s *hdr)
 {
 	if (timeout_watchdog != hdr->timeout_watchdog) {
 		cl_log(LOG_WARNING, "watchdog timeout: %d versus %d on this device",
-				(int)timeout_watchdog, (int)hdr->timeout_watchdog);
+				timeout_watchdog, (int)hdr->timeout_watchdog);
```

Comment:
hdr->timeout_watchdog is uint32_t, ideally use inttypes.h for the format specifier instead of casting

Reply:
Was leaning to go for a solution there that not only covers visualization but as well assignment to be processed further. Was thinking about basically wrapping the cast + e.g. checking it against MAX_LEGITIMATE_AGE for the assignments. We can use that wrapper for feeding to formatted printing as well.

Reply:
Initially wanted to cover checking timeouts coming from the disk-header including them being logged and converted into the in memory representation in this PR.
Doing this I again stumbled over the consistency check for headers coming from different disks that is actually always successful because the values read from the header are written to the in memory representation before they are being checked against exactly that.
So I guess checking for probability (what range of a timeout does actually make sense and when we should warn about or even reject a value), consistency (values coming from multiple disks and local config should match) and convert them to in memory representation (adjust byte-order, convert to int) should be split out into a new PR.